app/controllers/sessions_controller.rb

```diff
@@ -1,2 +1,30 @@
 class SessionsController < ApplicationController
+
+  def new
+  end
+
+  def create
+    @user = User.find_by(email: params[:session][:email])
+
+    unless @user.nil?
```

Comment:
Accoridng to the class style guide, we should not use unless with else, so this will need to be re-written: https://github.com/Ada-Developers-Academy/ruby-style-guide/blob/master/2-syntax.md#no-else-with-unless

We can still keep the specific messages, and you can do that with an `elsif`.